src/TorchSharp/PInvoke/LibTorchSharp.THSTensor.cs

```diff
@@ -992,6 +1020,9 @@ internal static partial class LibTorchSharp
         [DllImport("LibTorchSharp")]
         internal static extern IntPtr THSTensor_outer(IntPtr input, IntPtr vec2);
 
+        [DllImport("LibTorchSharp")]
+        internal static extern IntPtr THSTensor_ormqr(IntPtr input, IntPtr tau, IntPtr other, bool left, bool transpose);
```

Comment:
do we need to add the attribute `[MarshalAs(UnmanagedType.U1)]` to the bool parameters?

Reply:
Yes! Thank you.

Reply:
Do you know if it needs to go on the 'bool' return types, too, or just arguments?

Reply:
Yes, need to apply it to the return value too. Here is a good [example](https://learn.microsoft.com/en-us/visualstudio/code-quality/ca1414?view=vs-2022&tabs=csharp#example).